config.js

```diff
@@ -59,6 +65,71 @@ const CONFIG = {
   // DO NOT SET THIS TO TRUE IN PRODUCTION!
   [CONST.CONFIG.TELETEXT_VIEWER_HTTPS_REJECT_UNAUTHORIZED]: false,
 
+
+  // service definitions, in the following format:
+  //   [id_of_service]: {
+  //     // display name of service
+  //     name: str,
+  //
+  //     // url should be protocol independent (start with //), so that http and https can both be accommodated)
+  //
+  //     // To use this local teletext server, set `url` to '//localhost'
+  //     // (also ensure that `port` matches one of the enabled
+  //     //  TELETEXT_VIEWER_SERVE_HTTPS_PORT or TELETEXT_VIEWER_SERVE_HTTPS_PORT above)
+  //     url: str,
+  //     port: int,
+  //
+  //     // (optional) a SVN repository containing the service's pages
+  //     updateUrl: str,
+  //   }
+  [CONST.CONFIG.SERVICES_AVAILABLE]: {
+    [CONST.SERVICE_TEEFAX]: {
+      name: 'Teefax',
+      url: '//www.xenoxxx.com',
+      port: 80,
+
+      updateUrl: 'http://teastop.plus.com/svn/teletext/',
+    },
+
+    [CONST.SERVICE_DIGITISER]: {
+      name: 'Digitiser',
+      url: '//www.xenoxxx.com',
+      port: 80,
+
+      updateUrl: 'http://teastop.plus.com/svn/digitiser2k/',
+    },
+
+    [CONST.SERVICE_ARCHIVE]: {
+      name: 'Archive',
+      url: '//www.xenoxxx.com',
+      port: 80,
+
+      // @todo: does this have a SVN repo?
```

Comment:
@peterkvt80 does this have a SVN repo?

I'd also like to add Artfax and potentially other services here, do they have their own SVN or Git repos?

Reply:
Muttlee pages don't have a repo because they rely on the filename to contain the page number. If they were in the repo it would be possible to add pages that can't get loaded. If the pages are based on a repo, I run a script a few times a day that copies the files from the repo to /var/www and renames them. There are lots of problems with this approach so there is plenty of scope for improvement. I should really make a repo out of the Artfax and WTF folders.